spec/cli/navigate_spec.rb

```diff
@@ -54,6 +54,52 @@ def require_relative(path)
     end
   end
 
+  context "finds the right API", :vcr do
+    def stub_results(real_url)
+      api_fixtures_path = File::join(File::dirname(__FILE__), '..', 'fixtures', 'sample_api')
+      sample_api = File.read(api_fixtures_path)
+      stub_const('Razor::CLI::Navigate::PRIORITY_RAZOR_API', 'https://bad-priority:8151/api')
+      stub_const('Razor::CLI::Navigate::DEFAULT_RAZOR_API', 'https://bad-default:8151/api')
+      ::WebMock.stub_request(:get, real_url).to_return(body: sample_api, headers: {"Content-Type" => 'application/json'})
+      ::WebMock.stub_request(:head, real_url)
+    end
+    let(:cli_string) do
+      %w[--version].freeze
+    end
+    it "chooses the -u argument first" do
+      url = 'http://u-argument:8000'
+      stub_results(url)
+      cli_string = %W[-u #{url} --version].freeze
+      nav = Razor::CLI::Parse.new(cli_string).navigate
+      nav.last_url.to_s.should == url
+    end
+    it "chooses the ENV variable second" do
+      url = 'http://env-variable:8000'
+      stub_results(url)
+      ENV::store('RAZOR_API', url)
+      nav = Razor::CLI::Parse.new(cli_string).navigate
+      nav.last_url.to_s.should == url
+    end
+    it "chooses the priority URL third" do
+      url = 'http://priority-url:8000'
+      stub_results(url)
+      stub_const('Razor::CLI::Navigate::PRIORITY_RAZOR_API', url)
```

Comment:
Why not skip this and used the stubbed values above?

Reply:
The stubbed values above are intended to be invalid values only. Using a new value here indicates that this should be successful.